### PR TITLE
Added Documentation for IBM Quantum API 

### PR DESCRIPTION
**Pull request by iglesias-cardinale**
I added an actual markdown file for what I put in issue #26. This should be useful for anyone who is having trouble connecting their code to the IBM Quantum Computer.

### CODE REVIEW

File: IBM Quantum API Information.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+## Make sure your IBM Quantum API is activated
+
+In order to run code on IBM Quantum, we need to connect our code to the IBM Quantum service. This can be accomplished by activating your IBM Quantum API token. In lay terms, an API, or Application Programming Interface, is a bridge that connects two programs. It allows one program to request functions/data from another. In our present case, we want to connect our code written with qiskit to the IBM Quantum systems so that we can implement Bells Inequality on an actual quantum computer.
+
+This can be done by including the line 
+
+"**QiskitRuntimeService.save_account(token="YOUR_API_TOKEN", channel="ibm_quantum")**
```
Comment (laserlab) [inline IBM Quantum API Information.md]: Prettier markdown would be to use a code markup such as:
```markdown

> ```python
> QiskitRuntimeService.save_account(token="YOUR_API_TOKEN", channel="ibm_quantum")
> ```

```

File: .devcontainer/qiskit/devcontainer.json
Comment (laserlab) [inline .devcontainer/qiskit/devcontainer.json:1]: Looks like this should not be part of the PR. Please remove

**Review by laserlab — CHANGES_REQUESTED** [review]
- remove devcontainer.json
 - better markdown code formatting

**Comment by laserlab** [comment]
add label and milestone to PR

**Review by laserlab — APPROVED** [review]
looks good now

**Comment by iglesias-cardinale** [comment]
I'm glad this worked out. I basically made an entirely new branch in my forked repository and then tried to merge which I think was unnecessary.

**Pull request merged**